CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## 1.0.0  - 2023/05/01
+## 1.0.1 - 2023/02/02
+* 🚀 [FEATURE] Allow configuring custom array-like classes to be treated as collections when serializing.  More details can be found [here](https://github.com/blueprinter-ruby/blueprinter/pull/6). Thanks to [@toddnestor](https://github.com/toddnestor).
+* 💅 [ENHANCEMENT] Reduce object allocations in fields calculations to save some memory.  More details can be found [here](https://github.com/blueprinter-ruby/blueprinter/pull/5). Thanks to [@nametoolong](https://github.com/nametoolong).
+
+## 1.0.0  - 2023/01/05
```

Comment:
I updated the date from `2023/05/01` to `2023/01/05` because all the other dates in this file are `YYYY/MM/DD` it looks like (looking at lines 11 and 14 for obvious ones were the date is higher than 12).